src/client/app/effects/snippet.ts

```diff
@@ -288,6 +288,13 @@ export class SnippetEffects {
             case Snippet.ImportType.SAMPLE:
             case Snippet.ImportType.URL:
             case Snippet.ImportType.GIST:
+                let viewUrlMatch = new RegExp(`^${environment.current.config.editorUrl}\/#\/view`);
```

Comment:
It seems like there is a lodash method to escape a regular expression. We can probably just use this one:
_.escapeRegExp